src/Tonic/SampleTable.h

```diff
@@ -98,6 +114,16 @@ namespace Tonic {
     void resample(unsigned int frames, unsigned int channels){
       obj->resample(frames, channels);
     }
+      
```

Comment:
@andik  This work on SampleTable is unrelated to the cmake functionality, and probably doesn't belong in a pull request titled "Adding CMake functionality".